src/uct/base/uct_worker.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
```

Comment:
this one does not need extra copyright since it's copied from uct_md.h which has only this one